src/fuzz_main.cpp

```diff
@@ -102,20 +101,6 @@ int fuzz_main(const char* data, size_t sz)
   if (err)
     return err;
 
-  // Limit node size
-  uint64_t max_ast_nodes = DEFAULT_NODE_MAX;
```

Comment:
the define of DEFAULT_NODE_MAX can also be removed. But is it the case that sometimes this fuzzing test sets this ENV var and sometimes it doesn't?

Reply:
The use of the variable is emphasized in the docs, I don't think fuzzing should have a different default behavior than the program. I've dropped the #define.